examples/IBFE/explicit/ex5/example.cpp

```diff
@@ -117,6 +117,9 @@ struct TetherData
     }
 };
 
+bool use_boundary_mesh 			  = false;
+bool compute_fluid_traction 	  = false;
```

Comment:
Please run clang-format on your code before we continue the review.

Reply:
Do you see these spaces in the *.h and *.cpp files too? I think the version of clang-format I used did not fix the input files. Does the pre-commit hook automatically do that for all files?